dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -199,11 +160,31 @@ RUN cd {{$beatHome}}/.node \
      esac \
   && mkdir -p node \
   && curl ${NODE_DOWNLOAD_URL} | tar -xJ --strip 1 -C node \
-  && chmod ug+rwX -R $NODE_PATH \
-  && npm i -g -f @elastic/synthetics && chmod ug+rwX -R $NODE_PATH
-{{- end }}
+  && chmod ug+rwX -R $NODE_PATH
 
+# Install synthetics as a regular user, installing npm deps as root odesn't work
+RUN chown -R {{ .user }} $NODE_PATH
+USER {{ .user }}
+# If this fails dump the NPM logs
+RUN npm i -g --loglevel verbose -f @elastic/synthetics@stack_release || sh -c 'tail -n +1 /root/.npm/_logs/* && exit 1'
```

Comment:
`stack_release` is something we did recently post 8.x I believe, Could you double check this once again how its done in 7.x versions and address it? Thanks.

Reply:
@andrewvc afaik we haven't defined a specific tag for minor branches, is that something we want to do? The way it's done in v7.17 is installing latest, as it was on v8 before this change